src/lint.rs

```diff
@@ -0,0 +1,35 @@
+use anyhow::{ensure, Context};
+use std::{env, process::Command};
+
+pub fn lint() -> anyhow::Result<()> {
+    // `bevy` lives in the same folder as `bevy_lint_driver`, so we can easily find it
+    // using the path of the current executable.
+    let driver_path = env::current_exe()
+        .context("Failed to retrieve the path to the current executable.")?
+        .parent()
+        .expect("Path to file must have a parent.")
+        .join("bevy_lint_driver");
+
+    ensure!(
+        driver_path.exists(),
+        "`bevy_lint_driver` must be installed."
```

Comment:
A user running into this might appreciate some more guidance.

Reply:
I added more information on where it was searching, but because the actual process for installing the driver is still uncertain, I hesitate to add anything more.